### PR TITLE
Add s3 as resource

### PR DESCRIPTION
**Pull request by therealmarv**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 *.sql.gz
+docker-compose.override.yml
```
Comment (karenc) [inline .gitignore]: I don't think `docker-compose.override.yml` is used anymore so going to remove.

File: concourse/extract-content-s3.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            - name: content
+          run:
+            path: sh
+            args:
+              - -c
+              - |
+                ls -lah mock-struct
+                cp mock-struct/ident_hash content/
+                bridge extract $(cat mock-struct/ident_hash) -o content/
+      - put: remote
```
Comment (karenc) [inline concourse/extract-content-s3.yml]: I'm going to add `inputs: [content]` there to be more explicit what exactly we're storing in s3.

**Review by karenc — APPROVED** [review]
Thanks for your hard work! :tada:  I'll make some small changes.

File: concourse/test-s3-dummydata.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    repository: 18fgsa/s3-resource-simple
+
+resources:
+- name: remote
+  type: s3-simple
+  source:
+    access_key_id: ((aws-access-key))
+    secret_access_key: ((aws-secret-key))
+    bucket: ((s3bucket))
+    region: us-east-2
```
Comment (mmulich) [inline concourse/test-s3-dummydata.yml]: You forgot the use `((s3region))`. 😉

**Pull request merged**